samples/unit-tests/series/datagrouping/demo.js

```diff
@@ -825,6 +825,76 @@ QUnit.test('When groupAll: true, group point should have the same start regardle
         chart.series[1].points[1].dataGroup.start,
         'When the groupAll: false, and new extremes don\t influence the group, the start should not be changed.'
     );
+
+    // Change the data set to check the group start,
+    // when the extremes overlap with the point x.
```

Comment:
It might be useful to add an issue ticket's number here in comment to indicate what issue you are testing.